### DISCUSSION TITLE
Welcome to stm32fxxx-hal-libraries Discussions!

### DISCUSSION THREAD

**Discussion by MaJerle** [MaJerle/stm32fxxx-hal-libraries#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by djsg** [comment]
Hi MaJerle,

I want to use your code on my PCB board (F411) to read/write to SD card. 
However, in stm32f4xx_conf.h, there is no option for F411 MCU. 
Can you advise how to enable F411 for compiling the code?

**Comment by EmbeddedLife615** [comment]
邮件已收到，我会尽快回复的！

**Reply by djsg** [reply]
You can write in Chinese? 👍
